dyno/panda/dyno_panel.py

```diff
@@ -221,7 +222,15 @@ def writeFile():
 throttle_slider = tk.Scale(window, from_=0, to=100, orient=tk.HORIZONTAL, bg=color_label)
 throttle_canvas.create_window(75, 45, window=throttle_slider)
 #---------------------
-
+def mcu_flag(event):
+    mcu_acknowledged = False
+def read_data():
```

Comment:
Please rename this function to something more appropriate. Maybe check_acknowledge( )?